### ISSUE TITLE
Analisis of references to strings only

### ISSUE THREAD

**Issue by fabianMendez** [radare/radare2#11793]
Finding references to strings is a really important/common part of reversing (in my opinion). So it will be cool to have an option that just find references to strings specially when you are reversing a more "real life" sample which is > 30 MB, because as far as I know `aaa` and `aar` are currently the only options to find those references, which in a library that I'm currently working, crash because it eats all the ram in no time (memleak?)

PD: English is obviously no my native language

Thanks

**Comment by radare** [comment]
memory usage will be heavily reduced when we update the version of SDB, probably before next week release.

the current xrefs storage is not memory efficient because focus on access performance. you can use /r to find references to one place, but maybe will be good to add /rs to find references to strings.

**Comment by Maijin** [comment]
Did you try with current version of radare2 from git?

**Comment by fabianMendez** [comment]
> Did you try with current version of radare2 from git?

I just updated my radare2 installation, ran `aaa` and the memory usage was significantly reduced

Thanks

**Comment by Maijin** [comment]
So can be closed?

**Comment by radare** [comment]
nope. i would prefer to have an /rs command for that instead. using esil and data analysis to identify if it points to a string
